Add `no-descending-specificity` line number to the referenced selecto… (#8666)
stylelint/stylelint: .changeset/thin-carrots-shop.md

---

> "stylelint": patch

Reviewer: ```suggestion
"stylelint": minor
```

---

> Added: `no-descending-specificity` line number to the referenced selector in message

Reviewer 2: ```suggestion
Added: line number of the anchoring selector to `no-descending-specificity`'s report message
```

It seems better to me; @ybiquitous what do you think?

Reviewer: @Mouvedia Your suggested message might be more accurate, but let's keep the format "*Added: `rule-name` ...*". The current one is sufficient.

Reviewer 2: I don't care if we don't use mine, but the current one makes no sense.
i.e. it's not adding the line number of `no-descending-specificity`
It needs to be emended.

Author: Is this option acceptable?

Added: `no-descending-specificity` report message includes line number of the anchoring selector

Reviewer: > Is this option acceptable?
> 
> Added: `no-descending-specificity` report message includes line number of the anchoring selector

Sounds good 👍🏼

Author: Done
